packages/edit-post/src/components/keyboard-shortcuts/index.js

```diff
@@ -64,7 +69,7 @@ function KeyboardShortcuts() {
 		} );
 
 		registerShortcut( {
-			name: 'core/edit-post/toggle-block-navigation',
+			name: 'core/edit-post/toggle-list-view',
```

Comment:
Changing the shortcut name is a breaking change, I believe we should just keep the old name.

Reply:
Oh, I wasn't aware of this! Will revert it to the old name ASAP!